Sazabi.rc

```diff
@@ -1834,10 +1834,10 @@ BEGIN
     CONTROL         "Shift",IDC_CHK_SHIFT,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,19,101,30,10
     CONTROL         "Ctrl",IDC_CHK_CTRL,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,56,101,27,10
     CONTROL         "Alt",IDC_CHK_ALT,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,90,101,25,10
-    CONTROL         "←",IDC_CHK_LEFT,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,122,101,23,10
-    CONTROL         "↑",IDC_CHK_UP,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,152,101,23,10
-    CONTROL         "→",IDC_CHK_RIGHT,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,182,101,23,10
-    CONTROL         "↓",IDC_CHK_DOWN,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,212,101,23,10
+    CONTROL         "´eft",IDC_CHK_LEFT,"Button",BS_AUTOCHECKBOX | WS_TABSTOP,122,101,23,10

```

Comment:
It seems that chars in diff are garbled.
Is this just a problem when viewing a diff in GitHub?

Reply:
If it is just a GitHub problem, the change looks good for me.

Reply:
It seems that garbled character is contained unexpectedly.